ios/FluentUI/Popup Menu/MSPopupMenuItemCell.swift

```diff
@@ -165,6 +165,10 @@ class MSPopupMenuItemCell: MSTableViewCell {
         titleLabel.textColor = isSelected ? MSColors.PopupMenu.Item.titleSelected : MSColors.Table.Cell.title
         subtitleLabel.textColor = isSelected ? MSColors.PopupMenu.Item.subtitleSelected : MSColors.Table.Cell.subtitle
 
-        _accessoryType = isSelected ? .checkmark : .none
+        if let item = item,
```

Comment:
to be consistent with the other lines of code, can this be merge to oneline ? (isSelected && item?.checkmarkVisbleOnSelect) ? .checkmark  : .none